spec/graalvm/AsciidoctorConvertWithGraalVM.java

```diff
@@ -0,0 +1,37 @@
+import org.graalvm.polyglot.Context;
+
+import org.graalvm.polyglot.Context;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class AsciidoctorConvertWithGraalVM {
+  public static void main(String[] args) {
+    Context context = Context.newBuilder("js").allowIO(true).build();
+    context.getPolyglotBindings().putMember("IncludeResolver", new IncludeResolver());
+    context.eval("js", "var IncludeResolver = Polyglot.import('IncludeResolver');");
+    context.eval("js", "load('./build/asciidoctor-graalvm.js')");
+    context.eval("js", "load('./spec/graalvm/asciidoctor-convert.js')");
+  }
+
+  public static class IncludeResolver {
+    public String read(String path) throws IOException, URISyntaxException {
+      URL url = this.getClass().getClassLoader().getResource(path);
```

Comment:
It's not so bad because the user can decide if the file should be resolved from the classpath